src/vm/amd64/pinvokestubs.S

```diff
@@ -0,0 +1,277 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information. 
+//
+
+.intel_syntax noprefix
+#include "unixasmmacros.inc"
+#include "asmconstants.h"
+
+PInvokeStubForHost_CALLEE_SCRATCH_SIZE = 20h
+
+PInvokeStubForHost_STACK_FRAME_SIZE = PInvokeStubForHost_CALLEE_SCRATCH_SIZE
+
+// 8 FP parameter registers
+PInvokeStubForHost_XMM_SAVE_OFFSET = PInvokeStubForHost_STACK_FRAME_SIZE
+PInvokeStubForHost_STACK_FRAME_SIZE = PInvokeStubForHost_STACK_FRAME_SIZE + 80h
+
+// Ensure that the new rsp will be 16-byte aligned.
+.if ((PInvokeStubForHost_STACK_FRAME_SIZE + 8) % 16) != 0
+PInvokeStubForHost_STACK_FRAME_SIZE = PInvokeStubForHost_STACK_FRAME_SIZE + 8
+.endif
+
+// Return address is immediately above the local variables.
+PInvokeStubForHost_RETURN_ADDRESS_OFFSET = PInvokeStubForHost_STACK_FRAME_SIZE
+PInvokeStubForHost_PARAM_REGISTERS_OFFSET = PInvokeStubForHost_RETURN_ADDRESS_OFFSET + 8
+
+NESTED_ENTRY PInvokeStubForHost, _TEXT, NoHandler
+        alloc_stack     PInvokeStubForHost_STACK_FRAME_SIZE
+        END_PROLOGUE
+
+        // spill args
+        mov             [rsp + PInvokeStubForHost_PARAM_REGISTERS_OFFSET +  0h], rdi
+        mov             [rsp + PInvokeStubForHost_PARAM_REGISTERS_OFFSET +  8h], rdx
```

Comment:
The PInvokeStubForHost* is not needed for CoreCLR. Its use is under FEATURE_INCLUDE_ALL_INTERFACES that will be always off for CoreCLR. It may be easier to just delete it than to try to fix it.